iroh/src/commands/sync.rs

```diff
@@ -1,124 +1,103 @@
+
+use crate::config::ConsoleEnv;
 
 use super::RpcClient;
 
 const MAX_DISPLAY_CONTENT_LEN: u64 = 1024 * 1024;
 
 #[allow(clippy::large_enum_variant)]
-#[derive(Debug, Clone, Parser)]
+#[derive(Subcommand, Debug, Clone)]
 pub enum Commands {
-    Author {
-        #[clap(subcommand)]
-        command: Author,
-    },
-    Docs {
+    /// Manage documents
+    Doc {
         #[clap(subcommand)]
-        command: Docs,
+        command: DocCommands,
     },
-    Doc {
-        id: NamespaceId,
+
+    /// Manage document authors
+    Author {
         #[clap(subcommand)]
-        command: Doc,
+        command: AuthorCommands,
     },
 }
 
 impl Commands {
-    pub async fn run(self, client: RpcClient) -> anyhow::Result<()> {
+    pub async fn run(self, client: RpcClient, env: ConsoleEnv) -> Result<()> {
         let iroh = Iroh::new(client);
         match self {
-            Commands::Author { command } => command.run(iroh).await,
-            Commands::Docs { command } => command.run(iroh).await,
-            Commands::Doc { command, id } => command.run(iroh, id).await,
-        }
-    }
-}
-
-#[derive(Debug, Clone, Parser)]
-pub enum Author {
-    List,
-    Create,
-}
-
-impl Author {
-    pub async fn run(self, iroh: Iroh) -> anyhow::Result<()> {
-        match self {
-            Author::List => {
-                let mut stream = iroh.list_authors().await?;
-                while let Some(author_id) = stream.try_next().await? {
-                    println!("{}", author_id);
-                }
-            }
-            Author::Create => {
-                let author_id = iroh.create_author().await?;
-                println!("{}", author_id);
-            }
-        }
-        Ok(())
-    }
-}
-
-#[derive(Debug, Clone, Parser)]
-pub enum Docs {
-    List,
-    Create,
-    Import { ticket: DocTicket },
-}
-
-impl Docs {
-    pub async fn run(self, iroh: Iroh) -> anyhow::Result<()> {
-        match self {
-            Docs::Create => {
-                let doc = iroh.create_doc().await?;
-                println!("created {}", doc.id());
-            }
-            Docs::Import { ticket } => {
-                let doc = iroh.import_doc(ticket).await?;
-                println!("imported {}", doc.id());
-            }
-            Docs::List => {
-                let mut stream = iroh.list_docs().await?;
-                while let Some(id) = stream.try_next().await? {
-                    println!("{}", id)
-                }
-            }
+            Self::Doc { command } => command.run(&iroh, env).await,
+            Self::Author { command } => command.run(&iroh, env).await,
         }
-        Ok(())
     }
 }
 
 #[derive(Debug, Clone, Parser)]
-pub enum Doc {
-    StartSync {
-        peers: Vec<PeerSource>,
+pub enum DocCommands {
+    /// Set the active document (only works within the Iroh console).
+    Switch { id: NamespaceId },
+    /// Create a new document.
+    Init {
```

Comment:
why "init" and not "new"?

Reply:
@b5 proposed `init`. I had `create` before. Personally not sure what the best is between `new`, `create`, `init`

Reply:
@b5 thoughts?